app/controllers/email_authentications/invitations_controller.rb

```diff
@@ -8,22 +8,36 @@ def new
   end
 
   def create
-    user = User.new(user_params)
-    authorize([:manage, :admin, user])
-
-    User.transaction do
-      super do |resource|
-        user.email_authentications = [resource]
-        user.save!
-
-        next if permission_params.blank?
-
-        permission_params.each do |attributes|
-          user.user_permissions.create!(attributes.permit(
-            :permission_slug,
-            :resource_id,
-            :resource_type
-          ))
+    if Flipper.enabled?(:new_permissions_system_aug_2020, current_admin)
+      raise UserAccess::NotAuthorizedError unless current_admin.can?(:manage, :facility)
```

Comment:
This will eventually be replaced by the new authorize method from @vkrmis's [PR](https://github.com/simpledotorg/simple-server/pull/1208).

Reply:
Why are we checking if the user can manage _facilities_?

Reply:
Hm, I think that's the best approximation we have no? We only have the idea of `manage` and `view`, roughly. So, even if you can manage a single facility, that's enough for you to invite folks over.

Reply:
Ohh that's right! We don't really have a concept of managing _users_ do we. Would a better approximation simply be - is the user a manager?

Reply:
Hm, then we'll have to ensure if they are `manager` or `power_user` or any other `manager-esque` permission. One could make a `manager?` function on user or something. But it felt like using a standard pattern for all pages is better: like `authorize`. I did consider a `manage_user` action, which is simply a collection of `access_levels` that are allowed to manage a user. But I thought to wait on that until I get on migrating the user/admin pages. That may fit best here.

Reply:
Oh great point. We want to ask if you can perform the _action_ "manage", not if you are the role "manager". Yup, this looks fine until we introduce some concept of managing users 👍